test/middlewares/test-api-authentication-required.js

```diff
@@ -101,9 +100,46 @@ describe('apiAuthenticationRequired', function() {
           .set('Authorization','Bearer ' + access_token )
           .expect(200,successResponse,done);
       });
+    });
+  });
 
+  it('should return error response if invalid oauth credentials are supplied', function(done) {
+    var app = helpers.createStormpathExpressApp({
```

Comment:
The setup of the Express app is redundant for each test case. Maybe extract it into a `beforeEach` function?